avocado-core/src/main/scala/org/bdgenomics/avocado/discovery/ReadExplorer.scala

```diff
@@ -78,6 +79,8 @@ class ReadExplorer(referenceObservations: RDD[Observation]) extends Explorer wit
         quals(readPos),
         mapq,
         negativeStrand,
+        firstOfPair,
+        readPos,
```

Comment:
I'd say though that it's easier to store it as you've done. We can always go back and change things if necessary.